coverage_point_calculator/src/hexes.rs

```diff
@@ -5,13 +5,40 @@ use rust_decimal_macros::dec;
 
 use crate::{BoostedHexStatus, RadioType, Result};
 
+/// Breakdown of points for a hex.
+///
+/// Example:
+///   Outdoor Wifi with 1-hex boosted 5x,
+///   Rank 2, Assignment AAA:
+///     SplitPoints {
+///       modeled: 16,
+///       base: 8,
+///       boosted: 32
+///     }
+///
+///   Rank 2 splits modeled points in half.
+///   Boost at 5x adds 32 points ( (8 * 5) - 8 )
+#[derive(Debug, Default, Clone)]
+pub struct HexPoints {
```

Comment:
Despite the doc comments, it is impossible to view this struct in the documentation

Reply:
nevermind, I see the function

Reply:
🪄 Struct is now public.